kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KyuubiApplicationManager.scala

```diff
@@ -125,7 +132,7 @@ object KyuubiApplicationManager {
             s"Relative path ${uri.getPath} is not allowed, please use absolute path.")
         }
 
-        if (!localDirAllowList.exists(uri.getPath.startsWith(_))) {
+        if (!(localDirAllowList :+ tempDirForUpload.toString).exists(uri.getPath.startsWith(_))) {
```

Comment:
seems not needed, we can check whether the resource is uploaded in `BatchJobSubmission`.

Reply:
Addressed. Reverted this method by removing tempDirForUpload.